app/build.gradle

```diff
@@ -95,41 +95,39 @@ android {
 }
 
 ext {
-    androidX = "1.0.2"
-    materialDesign = "1.1.0-alpha05"
+    androidX = "1.1.0"
+    materialDesign = "1.2.0-alpha05"
     architectureComponents = "1.1.1"
     architectureComponentsExtensions = "1.1.1"
-    androidKtx = "1.0.2"
-    fragmentKtx = "1.0.0"
-    constraintLayout = "2.0.0-beta1"
-    lifecycle = "2.1.0-alpha04"
-    room = "2.1.0"
-    workManager = "2.2.0"
+    androidKtxCore = "1.2.0"
+    fragmentKtx = "1.2.3"
+    constraintLayout = "2.0.0-beta4"
+    lifecycle = "2.2.0"
+    room = "2.2.5"
+    workManager = "2.3.4"
     legacySupport = "1.0.0"
-    espressoCore = "3.1.1"
-    coreTesting = "2.0.0"
-    androidTestRunner = "1.0.2"
-    testRunner = "1.1.1"
-    coroutines = "1.3.1"
+    coreTesting = "2.1.0"
+    testRunner = "1.2.0"
+    coroutines = "1.3.5"
     retrofitCoroutinesAdapter = "0.9.2"
-    webkit = "1.1.0"
+    webkit = "1.2.0"
+    referrerLibrary = "1.1.2"
```

Comment:
How can we test this?

Reply:
i'd recommend building the app and uploading to the Play Store ad hoc distribution page.

- delete the build.gradle line where we add the `debug` suffix
- upload your apk, and get a link
- email this link to yourself, but append on query params to set the referrer
- install the app, watch logcat to ensure the referrer is picked up

(I have manually done the above. If you want to do it too, I can send you the referrer params to add)